### ISSUE TITLE
Error: No latent factors are being inputted.

### ISSUE THREAD

**Issue by Shumin96** [jishnu-lab/SLIDE#27]
I have tried the example data, SLIDE went well. When I used my own bulk-RNA seq data, it initally went well. The input file went through _checkDataParams(input_params)_successfully. However, after a few hours of computing, it reported an error as follows:
Output stored in C:/out/0.01_0.5_out/plotInteractions.png
No id variables; using all as measure variables
Scale for x is already present.
Adding another scale for x, which will replace the existing scale.
Saving 4.24 x 1.81 in image
Error in calcZMatrix(x_std, all_latent_factors, x_path = NULL, lf_path = NULL,  : 
  No latent factors are being inputted.
![error](https://github.com/jishnu-lab/SLIDE/assets/90902207/ac646edc-ae55-40f7-aa55-8e94d4b23e17)
![kable](https://github.com/jishnu-lab/SLIDE/assets/90902207/b41cc5ad-fabb-495f-8be6-09861d748591)
![checkdata](https://github.com/jishnu-lab/SLIDE/assets/90902207/c0466697-96bf-428d-98c0-03e83dcbf56f)

How can I deal with it?

**Comment by Hanxi-002** [comment]
Hi,
Thank you for submitting a Github issue.  

Your YAML file contains variables "delta1" "delta2" "lambda1" "lambda2".
These are unknown parameters to SLIDE, please adjust them into a format as shown in the example YAML file. 
To summarize, when you read the variable "input_params", which should be a list format in R, "delta" and "lambda" should be an array. 

If the problem persists after this, please let us know. 
Thank you.

**Comment by Shumin96** [comment]
Hi,
The YAML file contains variables "delta1" "delta2" "lambda1" "lambda2", which is same as the example file in the pdf manual.
&nbsp;
After trials, I found it is probably due to the thresh_fdr parameter. When making it bigger, SLIDE went well.


Thanks again for your kind help.



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jishnu-lab/SLIDE"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月16日(星期二) 晚上8:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jishnu-lab/SLIDE] Error: No latent factors are being inputted. (Issue #27)





 
Hi,
 Thank you for submitting a Github issue.
 
Your YAML file contains variables "delta1" "delta2" "lambda1" "lambda2".
 These are unknown parameters to SLIDE, please adjust them into a format as shown in the example YAML file.
 To summarize, when you read the variable "input_params", which should be a list format in R, "delta" and "lambda" should be an array.
 
If the problem persists after this, please let us know.
 Thank you.

**Comment by Hanxi-002** [comment]
So glad you were able to run SLIDE successfully!

**Comment by aditisk** [comment]
Hi @Shumin96, I am having a similar issue and was wondering what value for "thresh_fdr" eventually worked for you? Thanks.
